config/environments/test.rb

```diff
@@ -20,6 +20,9 @@
   ENV['PER_PAGE'] = '10'
   ENV['FLIPFLOP_KEY'] = 'yoyo'
   ENV['ALEPH_HINT_SOURCE'] = 'https://fake.example.com/s/fake/getserial_mini.xml?dl=1'
+
+  ENV['SCAN_EXCLUSIONS']='Materials -- Standards -- United States -- Periodicals;Materials -- Standards -- United States -- Periodicals;Standards, Engineering;Standards, Engineering -- Periodicals'
```

Comment:
IME this is not adequate. The spacing and hyphenation of subjects are not consistent. (This is why in `Rainbows` I actually strip out all the subdivisions and use only the first part of the term, at the cost of some accuracy - it would have taken actual time for me to map the options for subdivision syntax and I wasn't going to do that for an Easter egg. But in this case we actually have to do it.)

Reply:
Did you see that this was a problem with data coming from Aleph only? This is only a feature for Aleph records Aleph uses controlled subject headings so your concern should be less, uh, concerning?

Reply:
Will confirm with Catalog team that we use controlled authorities for Subjects and report back.

Reply:
I have confirmation from our catalog team that Subject headings are under authority control and thus should remain as accurate as this expects them to be.